templates/OperationSnippet.stencil

```diff
@@ -10,18 +10,42 @@ public func {{ op.name }}(
     guard let url = self.url(forTemplate: urlTemplate) else { return }
 
     // Construct query
```

Comment:
Since every operation will have a query params section and a header params section, I think it makes a lot of sense to move these sections into their own `stencil` files and add them via composition.

Reply:
Make sense. Create separate stencil file for header and query param